src/components/Navigation.tsx

```diff
@@ -0,0 +1,74 @@
+import { useContext } from 'react';
+import { ToDoContext } from '../store/AppContext';
+import cn from 'classnames';
+
+export const Navigation: React.FC = () => {
+  const { dispatch, state } = useContext(ToDoContext);
+
+  const handleFilterAll = () => {
+    dispatch({ type: 'FILTER_All' });
+  };
+
+  const handleFilterCompleted = () => {
+    dispatch({ type: 'FILTER_COMPLETED' });
+  };
+
+  const handleFilterActive = () => {
+    dispatch({ type: 'FILTER_ACTIVE' });
+  };
```

Comment:
```suggestion
  const handleFilterValue = (filterValue) => {
    dispatch({ type: 'FILTER_CHANGE', payload: filterValue});
  };
```
it can be only one function and only one action to handle all filter changes